src/components/Api.js

```diff
@@ -0,0 +1,100 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.header = options.headers;
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+  }
+  return Promise.reject(`Error ${res.status}`);
+  }
```

Comment:
You need to format the markup to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`


```js

  _checkResponse(res) {
    if (res.ok) {
      return res.json();
    }
    return Promise.reject(`Error ${res.status}`);
  }
```